deploy.sh

```diff
@@ -96,10 +70,10 @@ popd
 echo "Deploying ${ACTION_NAME}..."
 if [ -e ~/.wskprops ]
 then
-    $WSK action update "${ACTION_NAME}" --kind nodejs:10 "dist/${ACTION}.zip" --web true
+    $WSK action update "${ACTION_NAME}" --kind nodejs:18 "dist/${ACTION}.zip" --web true
```

Comment:
Should we try for node v22 since that is current?

Reply:
no, too much added scope, this is already a huge jump to something that I know well. I didn't want to jump beyond that in this PR, it can be done later more easily now with a focus on just what node is changing